samples/.common/dep/jquery.binarytransport.js

```diff
@@ -0,0 +1,54 @@
+ /**
+ *
+ * jquery.binarytransport.js
+ *
+ * @description. jQuery ajax transport for making binary data type requests.
```

Comment:
Please add some change description to jquery.binarytransport.js so it is obvious that it 's modified.